apps/condo/domains/document/schema/Document.js

```diff
@@ -0,0 +1,97 @@
+/**
+ * Generated by `createschema document.Document 'organization:Relationship:Organization:CASCADE; property?:Relationship:Property:CASCADE; category:Relationship:DocumentCategory:CASCADE; file:File;'`
+ */
+const get = require('lodash/get')
+
+const { GQLError } = require('@open-condo/keystone/errors')
+const { historical, versioned, uuided, tracked, softDeleted, dvAndSender } = require('@open-condo/keystone/plugins')
+const { GQLListSchema, getById } = require('@open-condo/keystone/schema')
+
+const FileAdapter = require('@condo/domains/common/utils/fileAdapter')
+const { getFileMetaAfterChange } = require('@condo/domains/common/utils/fileAdapter')
+const access = require('@condo/domains/document/access/Document')
+const { ERRORS } = require('@condo/domains/document/constants')
+const { ORGANIZATION_OWNED_FIELD } = require('@condo/domains/organization/schema/fields')
+
+
+const DOCUMENT_FOLDER_NAME = 'document'
+const Adapter = new FileAdapter(DOCUMENT_FOLDER_NAME, false, true)
+const fileMetaAfterChange = getFileMetaAfterChange(Adapter)
+
+const createAndReadOnlyFieldAccess = {
+    read: true,
+    create: true,
+    update: false,
+    delete: false,
+}
+
+const Document = new GQLListSchema('Document', {
+    schemaDoc: 'Document with file attached to organization or property. It could be some kind of property documentation, inspection reports or other documents',
+    fields: {
+        organization: ORGANIZATION_OWNED_FIELD,
+        property: {
+            schemaDoc: 'Property to which the document is attached',
+            type: 'Relationship',
+            ref: 'Property',
+            kmigratorOptions: { null: true, on_delete: 'models.SET_NULL' },
```

Comment:
Deleting property should revolve in deleting documents I guess. This is important part, because with `SET_NULL` this can be interpreted as moving house-scoped docs to organization scope